securedrop/requirements/python3/develop-requirements.txt

```diff
@@ -389,58 +432,66 @@ mypy==0.761 \
     --hash=sha256:a6bd44efee4dc8c3324c13785a9dc3519b3ee3a92cada42d2b57762b7053b49b \
     --hash=sha256:c6d27bd20c3ba60d5b02f20bd28e20091d6286a699174dfad515636cb09b5a72 \
     --hash=sha256:e2bb577d10d09a2d8822a042a23b8d62bc3b269667c9eb8e60a6edfa000211b1 \
-    --hash=sha256:f97a605d7c8bc2c6d1172c2f0d5a65b24142e11a58de689046e62c2d632ca8c1 \
-    # via -r requirements/python3/develop-requirements.in, sqlalchemy-stubs
+    --hash=sha256:f97a605d7c8bc2c6d1172c2f0d5a65b24142e11a58de689046e62c2d632ca8c1
+    # via
+    #   -r requirements/python3/develop-requirements.in
+    #   sqlalchemy-stubs
 netaddr==0.7.19 \
     --hash=sha256:38aeec7cdd035081d3a4c306394b19d677623bf76fa0913f6695127c7753aefd \
-    --hash=sha256:56b3558bd71f3f6999e4c52e349f38660e54a7a8a9943335f73dfc96883e08ca \
-    # via -r ../admin/requirements-ansible.in, -r requirements/python3/develop-requirements.in
+    --hash=sha256:56b3558bd71f3f6999e4c52e349f38660e54a7a8a9943335f73dfc96883e08ca
+    # via
+    #   -r ../admin/requirements-ansible.in
+    #   -r requirements/python3/develop-requirements.in
 nodeenv==1.3.3 \
-    --hash=sha256:ad8259494cf1c9034539f6cced78a1da4840a4b157e23640bc4a0c0546b0cb7a \
+    --hash=sha256:ad8259494cf1c9034539f6cced78a1da4840a4b157e23640bc4a0c0546b0cb7a
     # via pre-commit
 packaging==16.8 \
     --hash=sha256:5d50835fdf0a7edf0b55e311b7c887786504efea1177abd7e69329a8e5ea619e \
-    --hash=sha256:99276dc6e3a7851f32027a68f1095cd3f77c148091b092ea867a351811cfe388 \
-    # via dparse, pytest, safety
+    --hash=sha256:99276dc6e3a7851f32027a68f1095cd3f77c148091b092ea867a351811cfe388
+    # via
+    #   dparse
+    #   pytest
+    #   safety
 paramiko==2.6.0 \
     --hash=sha256:99f0179bdc176281d21961a003ffdb2ec369daac1a1007241f53374e376576cf \
-    --hash=sha256:f4b2edfa0d226b70bd4ca31ea7e389325990283da23465d572ed1f70a7583041 \
+    --hash=sha256:f4b2edfa0d226b70bd4ca31ea7e389325990283da23465d572ed1f70a7583041
     # via molecule
-pathlib2==2.3.5 ; python_version < "3.6" \
```

Comment:
Do we really not need pathlib2 any more?

Reply:
We do import it unconditionally in https://github.com/freedomofpress/securedrop/blob/develop/securedrop/requirements/python3/test-requirements.in#L8 , it might make sense to open a follow up to track removal since the changes here are unblocking CI